src/content/self-hosted/install/releases.mdx

```diff
@@ -25,6 +25,11 @@ id: releases
 - Oketo Self-Hosted will now show the [installation setup guide](administration/dashboard/#installation) by default
 - External resources can now be destroyed from the Okteto UI
 - External resources now supports [additional icons](reference/manifest.mdx##icon-string-optional)
+- Ingresses are now served through two ingress-controllers instances, enabling flexible configuration and improving connection stability.
+  - `ingress-nginx` will continue to be the main gateway for all inbound traffic to Okteto, but will stop processing individual users' ingresses. Instead it will forward users' ingresses to the new `okteto-nginx`.
+  - `okteto-nginx` sits behind `ingress-nginx` and it process exclusively users' ingresses.
+  - For additional configuration of both instances, please check the [ingress-controller configuration section](self-hosted/administration/configuration.mdx#nginx-ingress-controller).
+- Okteto Ingress default backend is now deployed separately of `ingress-nginx` or `okteto-nginx`.
```

Comment:
Should we include a link to the section where we explain the things that can be configured for the default-backend?